webapp/views/App/views/Users/UserInvite/UserInvite.js

```diff
@@ -11,51 +9,47 @@ import { useI18n } from '@webapp/store/system'
 import { FormItem, Input } from '@webapp/components/form/input'
 import DropdownUserGroup from '@webapp/loggedin/modules/users/components/dropdownUserGroup'
 
-import * as UserInviteState from './userInviteViewState'
-import { updateUserInviteProp, inviteUser, resetUserInviteState } from './actions'
+import { useInviteUser } from './store/hooks'
 
-const UserInviteView = () => {
-  useEffect(() => {
-    // Reset state on unmount
-    return () => dispatch(resetUserInviteState())
-  }, [])
+const UserInviteComponent = () => {
+  const { userInvite, onUpdateUserInvite, onInvite } = useInviteUser()
```

Comment:
since this is a local store, on onUpdateUserInvite can be shorten to ```onUpdate```. what do u think?